public/js/game.js

```diff
@@ -123,9 +139,18 @@ interact('.play-color').dropzone({
         getLevelData(level);
```

Comment:
Correct me if I'm wrong, but on what I'm looking at it says the next line is getLevelData(level).

Reply:
that's line 139, so it's not showing you every line of code.  Click the blue bar in between.